i6_models/parts/lstm.py

```diff
@@ -0,0 +1,70 @@
+__all__ = ["LstmBlockV1Config", "LstmBlockV1"]
+
+from dataclasses import dataclass
+import torch
+from torch import nn
+from typing import Dict, Tuple, Union
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class LstmBlockV1Config(ModelConfiguration):
+    input_dim: int
+    hidden_dim: int
+    num_layers: int
+    bias: bool
+    dropout: float
+    bidirectional: bool
+    enforce_sorted: bool
+
+    @classmethod
+    def from_dict(cls, model_cfg_dict: Dict[str, Any]):
+        model_cfg_dict = model_cfg_dict.copy()
+        return cls(**model_cfg_dict)
+
+
+class LstmBlockV1(nn.Module):
+    def __init__(self, model_cfg: Union[LstmBlockV1Config, Dict[str, Any]], **kwargs):
+        """
+        Model definition of LSTM block. Contains single lstm stack and padding sequence in forward call.
```

Comment:
Also add the "including dropout, batch-first variant,  hardcoded to use B,T,F input" part please.

Reply:
Could also add the supports scripting part.